# Текущая фаза миссии: "IDLE", "TAKEOFF", "DEPLOY", "ATTACK", "RETURN"
string phase

# Географический центр роя (Хаб)
float64 center_lat
float64 center_lon
float64 center_alt

# Время начала миссии (для синхронизации)
float64 start_time
